src/screencast/wlr_screencast.c

```diff
@@ -673,6 +673,7 @@ void xdpw_wlr_screencopy_finish(struct xdpw_screencast_context *ctx) {
 	struct xdpw_screencast_instance *cast, *tmp_c;
 	wl_list_for_each_safe(cast, tmp_c, &ctx->screencast_instances, link) {
 		cast->quit = true;
+		xdpw_pwr_stream_destroy(cast);
```

Comment:
This shouldn't be necessary, because destroying the stream is done with [xdpw_screencast_instance_destroy](https://github.com/emersion/xdg-desktop-portal-wlr/blob/master/src/screencast/screencast.c#L76) and xdpw_wlr_screencopy_init doesn't touch pipewire, so xdpw_wlr_screencopy_finish shouldn't. 

Thing is xdpw_screencast_instance_destroy is only called from [xdpw_wlr_frame_free](https://github.com/emersion/xdg-desktop-portal-wlr/blob/master/src/screencast/wlr_screencast.c#L38) which needs a roundtrip through the screencast loop. The pipewire stream is created after the wlroots buffer loop is started, so xdpw_wlr_frame_free should be reached.